### PR TITLE
Reference 'Divio Cloud' instances replaced by 'Divio'

### PR DESCRIPTION
**Pull request by mebdivio**

### CODE REVIEW

File: reference/divio-cli.rst [hunk 1]
```diff
@@ -244,7 +244,7 @@ directory.
 .. _divio-project-pull:
 
 ``pull``
-    Pulls the database or media files from the Divio Cloud.
+    Pulls the database or media files from Divio.
```
Comment (evildmp) [inline reference/divio-cli.rst:247]: _from the Divio cloud environment_

File: reference/divio-cli.rst [hunk 2]
```diff
@@ -253,14 +253,14 @@ directory.
 .. _divio-project-push:
 
 ``push``
-    Pushes the database or media files to the Divio Cloud.
+    Pushes the database or media files to Divio.
```
Comment (evildmp) [inline reference/divio-cli.rst:256]: _to the Divio cloud environment_

File: reference/divio-components-shell.rst [hunk 3]
```diff
@@ -60,7 +60,7 @@ containers.
 **Processes** - each session is isolated from any other extant processes (web processes, workers,
 other shell sessions).
 
-**Caches** - if your site's cache relies on the database (the Divio Cloud Projects default) then
+**Caches** - if your site's cache relies on the database (the Divio Projects default) then
```
Comment (evildmp) [inline reference/divio-components-shell.rst:63]: _the Divio Projects default_ > _the default in Divio projects_

File: reference/divio-cli.rst (hunk 1 above)
Comment (mebdivio) [reply reference/divio-cli.rst:247]: Done

File: reference/divio-cli.rst (hunk 2 above)
Comment (mebdivio) [reply reference/divio-cli.rst:256]: Done

File: reference/divio-components-shell.rst (hunk 3 above)
Comment (mebdivio) [reply reference/divio-components-shell.rst:63]: Done

**Pull request merged**